homeassistant/components/mobile_app/__init__.py

```diff
@@ -133,3 +177,108 @@ async def async_remove_entry(hass, entry):
             await cloud.async_delete_cloudhook(hass, entry.data[CONF_WEBHOOK_ID])
         except cloud.CloudNotAvailable:
             pass
+
+
+# pylint: disable=invalid-name
+class iOSConfigView(HomeAssistantView):
+    """A view that provides the iOOS push categories configuration."""
```

Comment:
```suggestion
    """A view that provides the iOS push categories configuration."""
```
:)

Reply:
Damn I actually spotted that earlier and changed it in a detached head and never committed. BUT iOOOOOOS sounds much better!